instrumentation/mongo/mongo-testing/src/main/groovy/io/opentelemetry/instrumentation/mongo/testing/AbstractMongoClientTest.groovy

```diff
@@ -120,6 +124,29 @@ abstract class AbstractMongoClientTest<T> extends AgentInstrumentationSpecificat
     collectionName = createCollectionName()
   }
 
+  def "test create collection calling build twice"() {
+    when:
+    runUnderTrace("parent") {
+      createCollectionCallingBuildTwice(dbName, collectionName)
+    }
+
+    then:
+    assertTraces(1) {
```

Comment:
try putting the sleep before this line, but agree not worth a lot of time, thx for adding the test and verifying manually 👍